### PR TITLE
Am 460 invoking document stores download metadata api

### PR DESCRIPTION
**Pull request by shrivsksavit**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-460

### Change description ###
(SPIKE) Create new BEFTA based feature file for invoking document stores download metadata API	

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by shrivsksavit** [comment]
only below fine had changes for AM-460 : 
1> F-460.feature
2> F-460_Test_Data_Base.td.json
3> F-460_Test_Data_Base_Get_Dcoument.td.json
4> S-4601.td.json

File: src/functionalTest/resources/features/F-460/S-4601.td.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "_extends_": "F-460_Test_Data_Base",
+  "_guid_": "S-4601",
+  "specs": [
+    "an active Solicitor profile in CCD with a specific variation of ACLs on a case type",
+    "an active solicitor profile with another specific variation of ACLs on the same case type",
+    "an active Solicitor profile having full permissions on the same case type",
+    "is made by the privileged user who just created the case",
+    "contains values for all fields under the case type",
+    "does not contain document fields 1,3,6 for Solicitor 1 as it does not has read access on these fields",
+    "does not contain document fields 2,5,7 for Solicitor 3 as it does not has read access on these fields"
```
Comment (aashishaggarwal123) [inline src/functionalTest/resources/features/F-460/S-4601.td.json]: Can you please tidy up this file based upon the used elements.

Comment (shrivsksavit) [reply src/functionalTest/resources/features/F-460/S-4601.td.json]: done

File: build.gradle [hunk 2]
```diff
@@ -65,7 +65,13 @@ jacocoTestCoverageVerification {
   }
 }
 
-
+configurations {
+  integrationTestCompile.extendsFrom testCompile
+  integrationTestRuntime.extendsFrom testRuntime
```
Comment (nitinsahu1) [inline build.gradle]: We may not needed these 2 integration tasks as Integration tasks has serenity report configured now. These 2 line will conflict with serenity integration.

File: build.gradle [hunk 3]
```diff
@@ -79,17 +85,17 @@ sourceSets {
...
     }
     resources.srcDir file('src/integrationTest/resources')
   }
 
   smokeTest {
     java {
-      compileClasspath += main.output
-      runtimeClasspath += main.output
+      compileClasspath += main.output + test.output
+      runtimeClasspath += main.output + test.output
```
Comment (nitinsahu1) [inline build.gradle]: We are no longer using default gradle report for smoke tests hence "test.output" may not be required here.

**Pull request closed without merging**